api/Makefile

```diff
@@ -1,23 +1,37 @@
+BASE_PATH = $(shell pwd)
+BIN_PATH := $(BASE_PATH)/../bin
+
+export PATH := $(BIN_PATH):$(PATH)
+
 help:                 ## Display this help message.
 	@echo "Please use \`make <target>\` where <target> is one of:"
 	@grep '^[a-zA-Z]' $(MAKEFILE_LIST) | \
 		awk -F ':.*?## ' 'NF==2 {printf "  %-26s%s\n", $$1, $$2}'
 
-init:                 ## Fill ../vendor/ and install tools.
+init:                 ## Install tools.
 	# https://github.com/uber/prototool#installation
 	curl -L https://github.com/uber/prototool/releases/download/v1.10.0/prototool-$(shell uname -s)-$(shell uname -m) -o ./prototool-new
```

Comment:
can we install prototool using `go build` like below?

Reply:
Getting error
```
	github.com/golang/protobuf/jsonpb/jsonpb_test_proto: module github.com/golang/protobuf@latest found (v1.5.2), but does not contain package github.com/golang/protobuf/jsonpb/jsonpb_test_proto
```
https://github.com/uber/prototool/issues/559